packages/shared/rollup.utils.js

```diff
@@ -0,0 +1,15 @@
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+const dirname = fileURLToPath(new URL('.', import.meta.url));
+const pkg = JSON.parse(fs.readFileSync(path.resolve(dirname, 'package.json')));
+
+export const aliasEntries = Object.fromEntries(
+  Object.entries(pkg.exports)
+    .filter(([, value]) => value.endsWith('.ts'))
+    .map(([key, value]) => [
+      key.replace(/^\./u, '@h5web/shared'),
+      value.replace(/^\.\/src\/(.+)\.ts$/u, '../shared/dist-ts/$1.d.ts'),
```

Comment:
This means, for instance, that import `@h5web/shared/utils` is aliased to `../shared/dist-ts/utils.d.ts`.

Reply:
Worth putting as comment IMO

Reply:
The beauty of reg exps: what you don't write in code, you need to write it in comments :joy_cat: